test/Microsoft.Azure.Devices.Edge.Test/helpers/SasManualProvisioningFixture.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Test.Helpers
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Test.Common;
+    using NUnit.Framework;
+
+    public class SasManualProvisioningFixture : ManualProvisioningFixture
+    {
+        [OneTimeSetUp]
+        public async Task SasProvisionEdgeAsync()
+        {
+            await Profiler.Run(
+                async () =>
+                {
+                    using (var cts = new CancellationTokenSource(Context.Current.SetupTimeout))
+                    {
+                        CancellationToken token = cts.Token;
+                        DateTime startTime = DateTime.Now;
+
+                        EdgeDevice device = await EdgeDevice.GetOrCreateIdentityAsync(
+                        Context.Current.DeviceId,
```

Comment:
See (new) indent comment in `X509ManualProvisioningFixture`...

Reply:
Ahh I get it now :)